pandas/tests/io/test_stata.py

```diff
@@ -112,7 +123,7 @@ def read_dta(self, file):
     def read_csv(self, file):
         return read_csv(file, parse_dates=True)
 
-    @pytest.mark.parametrize("version", [114, 117])
+    @pytest.mark.parametrize("version", [114, 117, 118, 119, None])
```

Comment:
in the future might be worth .a fixture for this

Reply:
Agree.